### PR TITLE
Revamp issue template and provide verbiage on +1 for tracking interest

### PR DESCRIPTION
**Pull request by david-yu**
Changes proposed in this PR:
- Added thumbs up verbiage to the feature and bug report issue templates. Best viewed in RAW file format due to instructions that are embedded as comments, which will not be displayed when filing.

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Bats tests added
- [ ] CHANGELOG entry added (*HashiCorp engineers only, community PRs should not add a changelog entry*)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -5,14 +5,30 @@ labels: bug
 
 ---
 
-When filing a bug, please include the following headings if possible. Any example text in this template can be deleted.
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request. Searching for pre-existing feature requests helps us consolidate datapoints for identical requirements into a single place, thank you!
+* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
```
Comment (ndhanushkodi) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -5,14 +5,30 @@ labels: bug
...
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
```
Comment (ndhanushkodi) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -5,21 +5,30 @@ labels: enhancement
 
 ---
 
-Please search the existing issues for relevant feature requests, and use the reaction feature (https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to add upvotes to pre-existing requests.
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request. Searching for pre-existing feature requests helps us consolidate datapoints for identical requirements into a single place, thank you!
+* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
```
Comment (ndhanushkodi) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -5,21 +5,30 @@ labels: enhancement
...
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
```
Comment (ndhanushkodi) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]:
```suggestion
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -5,21 +5,30 @@ labels: enhancement
...
 
 #### Feature Description
 
-A written overview of the feature.
+<!--- A description what this feature is and how it addresses the problem you are having. Describe potential UX for the feature if possible. --->
 
 #### Use Case(s)
 
-Any relevant use-cases that you see.
+<!--- Use cases where this feature is applicable for Consul on Kubernetes (i.e. type of application, type of Consul Use case i.e. Service Mesh, Service Disovery) --->
```
Comment (ndhanushkodi) [inline .github/ISSUE_TEMPLATE/feature_request.md:30]:
```suggestion
<!--- Use cases where this feature is applicable for Consul on Kubernetes (i.e. type of application, type of Consul Use case i.e. Service Mesh, Service Disovery). --->
```

**Review by ndhanushkodi — APPROVED** [review]
These changes look amazing and I love the detail.

**Review by lkysow — APPROVED** [review]

**Pull request merged**